llama_index/graph_stores/nebulagraph.py

```diff
@@ -559,17 +608,37 @@ def refresh_schema(self) -> None:
             tag_name = tag.cast()
             tag_schema = {"tag": tag_name, "properties": []}
             r = self.execute(f"DESCRIBE TAG `{tag_name}`")
-            props, types = r.column_values("Field"), r.column_values("Type")
+            props, types, comments = (
+                r.column_values("Field"),
+                r.column_values("Type"),
+                r.column_values("Comment"),
```

Comment:
Just curious, is it possible for this line to fail if the graph has no comments?

Reply:
Great point, but as this query is a DDL read query, the fields are actually fixed, so we could assume the field exists anyway, The only possibilities are value typed in Empty or String :-D

Reply:
Awesome!